lint_test/avoid_final_with_getter_test.dart

```diff
@@ -0,0 +1,17 @@
+// ignore_for_file: type_annotate_public_apis, prefer_match_file_name, unused_local_variable
+
+/// Check final private field with getter fail
+/// `avoid_final_with_getter`
+
+class Fail {
+  // expect_lint: avoid_final_with_getter
+  final int _myField = 0;
+
+  int get myField => _myField;
+
+  int get myField2 => _myField + 1;
```

Comment:
I would say we need to create a separate class with such kind of getter to test we are not going to lint in this situation.

Reply:
I will take it all to work